src/legacy/btchip_helpers.c

```diff
@@ -211,23 +211,23 @@ void btchip_write_u32_le(unsigned char *buffer, unsigned long int value) {
 
 
 
-void btchip_public_key_hash160(unsigned char *in, unsigned short inlen,
-                               unsigned char *out) {
-    cx_ripemd160_t riprip;
-    unsigned char buffer[32];
-    cx_hash_sha256(in, inlen, buffer, 32);
-    cx_ripemd160_init(&riprip);
-    cx_hash(&riprip.header, CX_LAST, buffer, 32, out, 20);
-}
-
-void btchip_compute_checksum(unsigned char* in, unsigned short inlen, unsigned char * output) {
-    unsigned char checksumBuffer[32];
-    cx_hash_sha256(in, inlen, checksumBuffer, 32);
-    cx_hash_sha256(checksumBuffer, 32, checksumBuffer, 32);
-
-    PRINTF("Checksum\n%.*H\n",4,checksumBuffer);
-    os_memmove(output, checksumBuffer, 4);
-}
+// void btchip_public_key_hash160(unsigned char *in, unsigned short inlen,
```

Comment:
why comment if dead code?

Reply:
because generally I prefer to touch the `/legacy` folder as little as possible (that's the old codebase). The entire folder will be removed some time this year, anyway :)